ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -238,3 +238,32 @@ def test_tensorflow_negative(
         fn_tree="negative",
         x=np.asarray(x, dtype=input_dtype),
     )
+
+
+# log_sigmoid
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_tf.valid_float_dtypes, min_value=1.0, max_value=1.0
```

Comment:
I meant `small_value_safety_factor` and `large_value_safety_facto` and not `min_value` and `max_value`, since they will cause hypothesis to only generate examples containing a list of 1s